compiler/env/OMRCPU.cpp

```diff
@@ -97,9 +100,27 @@ OMR::CPU::initializeByHostQuery()
 
    }
 
+bool
+OMR::CPU::is(OMRProcessorArchitecture p)
+   {
+   if (TR::Compiler->omrPortLib == NULL)
+      {
+      TR_ASSERT_FATAL(false, "This version should not be called!\n");
+      return false; 
+      }
```

Comment:
How about:

```
TR_ASSERT_FATAL(TR::Compiler->omrPortLib != NULL, "Should not be calling this OMR level API without a valid port library pointer. Perhaps we did not initialize the port library properly?\n");
```

instead of the entire block. Similarly elsewhere. The assert should explain why we are asserting.